public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -97,15 +97,30 @@ export class PrometheusDatasource {
       activeTargets.push(target);
 
       var query: any = {};
-      query.expr = this.templateSrv.replace(target.expr, options.scopedVars, self.interpolateQueryExpr);
-      query.requestId = options.panelId + target.refId;
       query.instant = target.instant;
 
-      var interval = this.templateSrv.replace(target.interval, options.scopedVars) || options.interval;
+      var interval = this.intervalSeconds(options.interval);
```

Comment:
Please use https://github.com/grafana/grafana/blob/master/public/app/core/utils/kbn.js#L204 instead of `this.intervalSeconds`.

Reply:
Done.